### PR TITLE
Add a section on integrating with faker

### PR DESCRIPTION
**Pull request by cowboyd**
## Motivation

Faker is awesome, and it's really useful with GraphGen. At the very least, we need to provide a recipe on how it can be used with Faker.

### CODE REVIEW

**Review by taras — APPROVED** [review]
👍

File: www/docs/usage/faker.md [hunk 1]
```diff
@@ -2,5 +2,67 @@
...
+generation methods in the entire Faker library as and put them under
+the `@faker/` namespace, and second, we'll make a very common list of
+patterns for accessing those methods. Here's our faker generator:
+
+``` javascript
+import { faker as globalFaker, Faker } from "@faker-js/faker";
+import { createDispatch } from "@frontside/graphgen";
+
+const methods = Object.entries(globalFaker).reduce((methods, [name, mod]) => {
+  if (mod) {
```
Comment (wKich) [inline www/docs/usage/faker.md:34]: Does 'faker' really contain 'falsy' values?

Comment (cowboyd) [reply www/docs/usage/faker.md:34]: It's to filter out faker "modules" that don't have any generator content, so mostly there is `faker.internet`, and `faker.address` and `faker.datatype` etc... but there are also properties just "hanging" on `faker` that don't represent a collection of generators like `faker.fakerStatic`

Comment (cowboyd) [reply www/docs/usage/faker.md:34]: Although it does seem weird.....

**Pull request merged**